cicecore/cicedynB/infrastructure/io/io_netcdf/ice_restart.F90

```diff
@@ -661,6 +677,7 @@ subroutine read_restart_field(nu,nrec,work,atype,vname,ndim3, &
 
       character(len=*), parameter :: subname = '(read_restart_field)'
 
+#ifdef USE_NETCDF
```

Comment:
I *think* this `ifdef` is not needed, I don't see any direct calls to NetCDF subroutines in `read_restart_field`.

Reply:
I think you're right.  Every other subroutine in this file has the USE_NETCDF logic, so I might argue we should keep it here too for consistency and in case direct NetCDF calls are added later.  I guess I propose we keep it for now.